Code/Framework/AzToolsFramework/AzToolsFramework/ToolsComponents/EditorNonUniformScaleComponent.cpp

```diff
@@ -61,6 +61,8 @@ namespace AzToolsFramework
                     ->Attribute(AZ::Script::Attributes::Module, "editor")
                     ->Attribute(AZ::Script::Attributes::Scope, AZ::Script::Attributes::ScopeFlags::Automation);
             }
+
+            NonUniformScaleComponentMode::Reflect(context);
```

Comment:
Nitpicky suggestion: Most other places I've seen that have Reflect() functions calling other Reflect() functions put those calls at the top, not the bottom. Makes it a tiny bit easier to spot them.

Reply:
Solved